scripts/slack-release-notification.mjs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env zx
+
+import SlackNotify from 'slack-notify';
+import {config} from 'dotenv';
+import { createRequire } from "module"; 
+
+const require = createRequire(import.meta.url); 
+const json = require("../packages/components/package.json");
+
+const result = config({path: 'local.env'});
+const slack = SlackNotify(result.parsed.SLACK_NOTIFICATION);
+
+const message = ':information_source: New Release v' + json.version + ' :tada:';
+
+slack.send({
```

Comment:
something is wrong or not configured in prettier 🤔
wrong formatted

Reply:
Wie meinst du das?

Reply:
e.g. line 33-38 in your script
too much spaces and indentations

Reply:
these are some of my prettier settings
![image](https://user-images.githubusercontent.com/59875255/155686073-d9efb4d3-442c-4d7d-9957-b5fb6607def6.png)